internal/zinc-scripted/src/test/scala/sbt/internal/inc/IncHandler.scala

```diff
@@ -335,6 +335,7 @@ case class ProjectStructure(
     (incO, sco)
   }
   val exportPipelining = incOptions.pipelining
+  val noOuroboros = incProps.getProperty("noOuroboros", "false").toBoolean
```

Comment:
Haha. I had to look ouroboros up. It's very appropriate. The only bad thing is that I'm not sure how you would figure out what it does without looking at where it's used in the code.

Reply:
Yeah, it's opaque.  But it's also terse and unique, so I was happy enough with it.